MapboxCoreNavigation/RouteController.swift

```diff
@@ -158,6 +158,18 @@ open class RouteController: NSObject {
             NotificationCenter.default.post(name: .routeControllerDidReroute, object: self, userInfo: userInfo)
         }
     }
+    
+    /**
+     :nodoc:
+     A star rating from 0 to 5 where 0 is the worst and 5 is the best.
+     */
+    public var endOfRouteStarRating: Int?
+    
+    /**
+     :nodoc:
+     The users comment amount their navigation experience.
+     */
+    public var endOfRouteComment: String?
```

Comment:
Could we make this a public function to set both at once, so that you can't get the properties publicly? Something like `setEndOfRoute(rating: Int, comment: String?)`.

This way, if we wanted in the future we could control the conditions under which these properties could be set (e.g. only after enough of the route was completed, only if not set before, etc).